jobs/build-charms/builder_launchpad.py

```diff
@@ -105,14 +114,17 @@ def _lp_recipe(self):
             rec = self._lp.charm_recipes.new(
                 name=self._lp_recipe_name,
                 auto_build=False,
-                auto_build_channels=self.SNAP_CHANNEL,
+                auto_build_channels={},
                 build_path=self.opts.get("subdir", ""),
+                description=f"Recipe for {self._lp_project} {self._lp_branch}",
                 git_ref=ref,
                 owner=self._lp_owner,
                 project=self._lp_project,
                 store_channels=[],
                 store_upload=False,
             )
+        rec.auto_build_channels = self._lp_channels
+        rec.lp_save()
```

Comment:
update the `auto_build_channels` in the recipe (although we're not using the auto-build channels)